PlexRequests.UI/Views/Shared/Partial/_Navbar.cshtml

```diff
@@ -38,10 +39,11 @@
                 {
                     <li><a id="donate" href="https://www.paypal.me/PlexRequestsNet" target="_blank"><i class="fa fa-heart" style="color: red"></i> @UI.Layout_Donate</a></li>
                 }
+                <li id="customDonate" style="display: none"><a id="customDonateHref" href="https://www.paypal.me/PlexRequestsNet" target="_blank"><i class="fa fa-heart" style="color: yellow;"></i> <span id="donationText">Donate to Library Maintainer</span></a></li>
```

Comment:
Can we add the text into the Resources? Similar to what we have done above `@UI.Layout_Donate`